GiveNGo/GiveNGo
# Request page

## Description
* Built request page with input box for request, radio "buttons" to select type of request, and submit button to send request to DB and redirect user to dashboard

## Changes I made
* RequestPage.tsx updated with hooks, addRequest function to send new request to DB, input box, radio buttons, and styling

## How to Test
* `npm start`
* navigate through login and select "Request" on bottom navigation bar
* you should see:
![image](https://user-images.githubusercontent.com/60517001/88312875-8fb84d80-ccc7-11ea-9bad-770504ec0ad0.png)
